pkg/formatter/table.go

```diff
@@ -41,77 +41,82 @@ func FormatNodeReport(records []*reporting.NodeReportItem) FormattedResult {
 		return FormattedResult{"No nodes found to analyze.", ""}
 	}
 
-	termWidth, _, err := terminal.GetSize(int(os.Stdout.Fd()))
-	if err != nil {
-		termWidth = MinTermWidth
-	}
-	buffer := bytes.NewBufferString("")
+	var (
+		buffer = bytes.NewBufferString("\n-- REPORT SUMMARY --\n\n")
+		table  = tablewriter.NewWriter(buffer)
+	)
 
 	// Let's look at content to pre-determine the best column widths
-	table := tablewriter.NewWriter(buffer)
 	table.SetAutoWrapText(true)
 	table.SetReflowDuringAutoWrap(true)
 	table.SetHeader(NodeReportHeader)
-	table.SetAutoFormatHeaders(false) // Don't make our headers capitalized
-	table.SetRowLine(true)            // don't show row seps
+	table.SetAutoFormatHeaders(false) // don't make our headers capitalized
+	table.SetRowLine(false)           // don't show row seps
```

Comment:
I hav an incoming PR that will also display a table without separators since I want to change it to display a number not the actual list of cookbooks.